x-pack/test/api_integration/apis/ml/calendars/update_calendars.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { COMMON_REQUEST_HEADERS } from '../../../../functional/services/ml/common';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertestWithoutAuth');
+  const ml = getService('ml');
+
+  describe('update_calendars', function () {
+    before(async () => {
+      await ml.testResources.setKibanaTimeZoneToUTC();
+    });
+
+    const calendarId = `test_update_cal`;
+    const originalCalendar = {
+      calendar_id: calendarId,
+      job_ids: ['test_job_1'],
+      description: `Test calendar`,
+    };
+    const originalEvents = [
+      { description: 'event 1', start_time: 1513641600000, end_time: 1513728000000 },
+    ];
+
+    const updateCalendarRequest = {
```

Comment:
I'd suggest to name this `updateCalendarRequestBody`.

Reply:
Fixed in https://github.com/elastic/kibana/pull/72564/commits/121d1e5fdd860b91516a4ae3e2023cec45205c9b